### ISSUE TITLE
Failing to create datasets with " Queries of this type are not supported" error

### ISSUE THREAD

**Issue by leigh507** [aws-samples/aws-cudos-framework-deployment#135]
```
Creating dataset: customer_all...Checking if CUR is enabled and available...
        Athena table: 00000000_cur
        Resource IDs: yes
2022-01-07 17:09:43,876 - ERROR - cid.helpers.athena:execute_query:156 - Athena query failed: An error occurred (InvalidRequestException) when calling the StartQueryExecution operation: Queries of this type are not supported
2022-01-07 17:09:43,876 - ERROR - cid.helpers.athena:execute_query:157 - Full query: SELECT
  identity_line_item_id
FROM
  0000000000_cur
WHERE
  savings_plan_savings_plan_a_r_n NOT LIKE ''
LIMIT 10

Traceback (most recent call last):
  File "/home/cloudshell-user/.local/lib/python3.7/site-packages/cid/helpers/athena.py", line 147, in execute_query
    WorkGroup='primary'
  File "/home/cloudshell-user/.local/lib/python3.7/site-packages/botocore/client.py", line 391, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/home/cloudshell-user/.local/lib/python3.7/site-packages/botocore/client.py", line 719, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.errorfactory.InvalidRequestException: An error occurred (InvalidRequestException) when calling the StartQueryExecution operation: Queries of this type are not supported

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/cloudshell-user/.local/bin/cid-cmd", line 8, in <module>
    sys.exit(main())
  File "/home/cloudshell-user/.local/lib/python3.7/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/home/cloudshell-user/.local/lib/python3.7/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/home/cloudshell-user/.local/lib/python3.7/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/cloudshell-user/.local/lib/python3.7/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/cloudshell-user/.local/lib/python3.7/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/home/cloudshell-user/.local/lib/python3.7/site-packages/click/decorators.py", line 38, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "/home/cloudshell-user/.local/lib/python3.7/site-packages/cid/cli.py", line 41, in deploy
    App.deploy()
  File "/home/cloudshell-user/.local/lib/python3.7/site-packages/cid/common.py", line 208, in deploy
    self.create_datasets(required_datasets)
  File "/home/cloudshell-user/.local/lib/python3.7/site-packages/cid/common.py", line 487, in create_datasets
    if self.create_dataset(dataset_definition):
  File "/home/cloudshell-user/.local/lib/python3.7/site-packages/cid/common.py", line 527, in create_dataset
    '${cur_table_name}' else name for name in _views]
  File "/home/cloudshell-user/.local/lib/python3.7/site-packages/cid/common.py", line 527, in <listcomp>
    '${cur_table_name}' else name for name in _views]
  File "/home/cloudshell-user/.local/lib/python3.7/site-packages/cid/common.py", line 84, in cur
    print(f"\tSavingsPlans: {'yes' if _cur.hasSavingsPlans else 'no'}")
  File "/home/cloudshell-user/.local/lib/python3.7/site-packages/cid/helpers/cur.py", line 91, in hasSavingsPlans
    self._hasSavingsPlans = bool(len(self.athena.execute_ahq('hasSavingsPlans', **kwargs)))
  File "/home/cloudshell-user/.local/lib/python3.7/site-packages/cid/helpers/athena.py", line 254, in execute_ahq
    execution_id = self.execute_query(query)
  File "/home/cloudshell-user/.local/lib/python3.7/site-packages/cid/helpers/athena.py", line 159, in execute_query
    raise Exception
Exception
```

**Comment by leigh507** [comment]
Looks like the FROM clause is missing quotation marks around the table name

**Comment by darken99** [comment]
Did you try running this query directly in Athena console?

**Comment by leigh507** [comment]
Yes - i updated the cid/queries files locally and it worked a dream. I'll get a PR together shortly.

**Comment by darken99** [comment]
It will be fixed in #140
